Packs/Base/Scripts/DBotMLFetchData/DBotMLFetchData.py

```diff
@@ -155,6 +156,14 @@ class ShortTextException(Exception):
                      "tinyurl.ms", "tip.pe", "ty.by"}
 
 
+def hash_value(simple_value):
```

Comment:
plz use the has in common server python, better to use the same hash function as we used in the telemetry